build/dockerfiles/Dockerfile

```diff
@@ -11,13 +11,18 @@
 #
 
 FROM golang:1.13-alpine3.12 as builder
+ARG CGO_ENABLED=0
```

Comment:
how are these used?

Reply:
used by go test and go build tools. it is equivalent to `ENV`, **potentially**  used for the mutli-arch build.  Can be replaced with `ENV`

Reply:
I don't see it used there. I would assume something like `CGO_ENABLED=$CGO_ENABLED GOOS=$GOOS` on test and build lines.

Reply:
AFAIK it's environment variable. I assume it's not necessary. @metlos wdyt?

Reply:
ah ok, make sense

Reply:
replaced with ENV